### ISSUE TITLE
Crashing with exit code 0xC000001D (illegal instruction)

### ISSUE THREAD

**Issue by marticliment** [shilangyu/scoop-search#65]
## Description

I am the developer of UniGetUI, and UniGetUI uses scoop-search to parse scoop search packages. I have received a report of scoop-search crashing on an i5-2500k CPU with return code (-1073741795) 0xC000001D. (Illegal instruction)

Source issue: https://github.com/marticliment/UniGetUI/issues/2550

## Logs

Scoop Search verbose logs
```txt
I will try to obtain them as soon as I can.
```

Info on how scoop search was called:
```txt
Logged subprocess-based task on manager Scoop. Task type is FindPackages
Subprocess executable: "C:\Users\~\scoop\shims\scoop-search.exe"
Command-line arguments: "7zip"
Process start time: 7/28/2024 11:56:16 PM
Process end time:   7/28/2024 11:56:19 PM

Return code: FAILED (-1073741795)
```

**Comment by shilangyu** [comment]
What version of scoop-search are you using? This has been fixed in v1.5

**Comment by marticliment** [comment]
My bad, I will let them know
